stan/math/prim/fun.hpp

```diff
@@ -270,7 +271,6 @@
 #include <stan/math/prim/fun/read_corr_matrix.hpp>
 #include <stan/math/prim/fun/read_cov_L.hpp>
 #include <stan/math/prim/fun/read_cov_matrix.hpp>
-#include <stan/math/prim/fun/real.hpp>
```

Comment:
Don't remove `real()` and `imag()`.  We won't use those functions by that name in Stan, but we need it to support programs written against the C++ standard library for our autodiff.  

You can just have `get_real()` and `get_imag()` delegate to `real()` and `imag()` for implementation, but the generic form you have below is probably easier.